lib/ansible/modules/system/mount.py

```diff
@@ -131,14 +137,14 @@
     state: present
 '''
 
-import os
```

Comment:
import os belongs here rather than mixed in with the ansible imports because it's from the stdlib (group 1 in pep8) while the ansible imports are application specific imports (group 3)